natlas-server/app/scope/ipscanmanager.py

```diff
@@ -1,50 +1,84 @@
 from netaddr import IPSet, IPNetwork
-from app.cyclicprng import CyclicPRNG
+from app.scope.cyclicprng import CyclicPRNG
 
 
 class IPScanManager:
 	networks = []
 	total = 0
 	rng = None
 
+	def __init__(self):
+		self.networks = []
+		self.total = 0
+		self.rng = None
+
 	def __init__(self, whitelist, blacklist):
-		ipset = IPSet([])
 		self.networks = []
 		self.total = 0
 		self.rng = None
 
+		self.setWhitelist(whitelist)
+		self.setBlacklist(blacklist)
+		self.initializeManager()
+
+	def setWhitelist(self, whitelist):
```

Comment:
I'm trying to move more towards [snake_case](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables) method names, even though they don't seem to be enforced inside classes. Can you update these files to use the new method name style?